app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-Root view
\ No newline at end of file
+<ul>
+  <% @users.each do |user| %>
+    <h4><%= user.id %></h4>
+    <li><h1><%= user.name %></h1></li>
+    <h4>Number of posts: <%= user.posts.count %></h4>
+  <% end %>
+</ul>
```

Comment:
- [ ] Great job creating views for your app. However, there're few things to do before you could move on to the next milestone.
- [ ] It is required that you index view page should match the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAACJF5B2AX2L63CTAV4ZLMVOGZKD6ETQ). :point_down: 

![username](https://github.com/Gh-Reza/BlogApp/assets/26597432/ae037b3f-2d55-4ab1-961f-62fb4130bbb3)

- After creating two users and starting your app, this is how your view page displayed. :point_down: 

![your-page](https://github.com/Gh-Reza/BlogApp/assets/26597432/e46451ce-e996-40bf-a985-e90751a15d63)